### PR TITLE
Add mainnet to cube 

### PR DESCRIPTION
**Pull request by cppfuns**

### CODE REVIEW

**Comment by cppfuns** [comment]
@ligi Our project is in urgent need of this review. Please give priority to it. Thank you for your support

File: _data/chains/eip155-1818.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "icon": "cube",
+  "rpc": [
+    "https://http-mainnet.cube.network",
+    "wss://ws-mainnet.cube.network",
+    "https://http-mainnet-sg.cube.network",
+    "wss://ws-mainnet-sg.cube.network",
+    "https://http-mainnet-us.cube.network",
+    "wss://ws-mainnet-us.cube.network"
+  ],
+  "nativeCurrency": {
```
Comment (ligi) [inline _data/chains/eip155-1818.json:13]:
```suggestion
  ],
  "faucets": [],
  "nativeCurrency": {
```

**Review by ligi — CHANGES_REQUESTED** [review]
The faucets field is missing - please apply commit

**Comment by ligi** [comment]
Please consider contributing to our grant: https://gitcoin.co/grants/2974/ethereum-lists

**Pull request merged**